client/src/lib/schemaFileUtils.ts

```diff
@@ -58,7 +58,7 @@ export function replaceFileData(
   return innerReplaceFileDataWithURL(schema, schema.$defs, obj, replacer) as Record<string, unknown>;
 }
 
-export type FileFormat = 'audio' | 'image' | 'text' | 'document';
+export type FileFormat = 'audio' | 'image' | 'text' | 'document' | 'universal';
```

Comment:
Euh, why ? 
That's not one we support in the backend and will lead to errors no ?

Maybe `undefined` ? or you should strip it before sending it with the schema ?